kits/beta-base/scripts/deploy_deps.js

```diff
@@ -3,13 +3,13 @@ const fs = require('fs')
 
 const namehash = require('eth-ens-namehash').hash
 
-const deploy_ens = require('@aragon/os/scripts/deploy-beta-ens.js')
-const deploy_apm = require('@aragon/os/scripts/deploy-beta-apm.js')
+const deploy_ens = require('@aragon/os/scripts/deploy-test-ens.js')
+const deploy_apm = require('@aragon/os/scripts/deploy-apm.js')
 const deploy_id = require('@aragon/id/scripts/deploy-beta-aragonid.js')
 
 // ensure alphabetic order
 const apps = ['finance', 'token-manager', 'vault', 'voting']
-const appIds = apps.map(app => namehash(require(`@aragon/apps-${app}/arapp`).appName))
+const appIds = apps.map(app => namehash(require(`@aragon/apps-${app}/arapp`).environments.default.appName))
```

Comment:
In #35 I changed it to directly calculating the namehash by appending `.aragonpm.eth`. Can you merge master into this branch so the actual change can be seen?

Reply:
But what version do you want to keep? I'd rather have the one that still gets it from arapp, but as you want.